keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandler.kt

```diff
@@ -82,30 +79,6 @@ class ApplicationLoadBalancerHandler(
     return listOf(Task(id = taskRef.taskId, name = description))
   }
 
-  override suspend fun delete(resource: Resource<ApplicationLoadBalancer>) {
```

Comment:
Glad to see the delete methods removed.

Reply:
I didn’t remove the method from the base class entirely